kubernetes/clickhouse/templates/secret.yaml

```
@@ -0,0 +1,9 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: clickhouse-credentials
+type: Opaque
+stringData:
+  {{- range .Values.global.clickhouse.users }}
+  {{ .name }}_password_sha256_hex: "{{ .password | sha256sum }}"
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 0 but found 2

(indentation)